addons/mail/models/mail_activity.py

```diff
@@ -378,6 +378,95 @@ def _check_access_assignation(self):
     # ORM overrides
     # ------------------------------------------------------
 
+    @api.model
+    def _search(self, args, offset=0, limit=None, order=None, count=False, access_rights_uid=None):
```

Comment:
Just for ordering purpose, shouldn't we have an order like CRUD, search, read_group ? main ORM methods first, then "search methods", ... Just to be that annoying guy.